spring-petclinic-microservices/spring-petclinic-admin-server/pom.xml

```diff
@@ -16,12 +16,17 @@
       </parent>
 
     <properties>
-        <spring-boot-admin.version>2.2.2</spring-boot-admin.version>
+        <spring-boot-admin.version>2.5.5</spring-boot-admin.version>
```

Comment:
how about defining these versions in the parent pom.xml in dependency management?